app/controllers/verify/phone_controller.rb

```diff
@@ -9,7 +9,10 @@ def new
     end
 
     def create
-      if step.complete
+      result = step.submit
+      analytics.track_event(Analytics::IDV_PHONE_CONFIRMATION, result.to_h)
```

Comment:
I like the idea that the `track_event` method can assert that the object is_a `FormResponse` instance and so depend upon its API. The hash is more brittle and we're relying on convention instead of a formal compact. So yes, I would prefer to see `.to_h` inside as well, but we could do that on a follow-on PR since it represents a far-reaching change.